chainer/functions/array/hstack.py

```diff
@@ -53,11 +53,31 @@ def hstack(xs):
     """Concatenate variables horizontally (column wise).
 
     Args:
-        xs (list of chainer.Variable): Variables to be concatenated.
+        xs (list of :class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`):
+            Input variables to be concatenated. The variables must have the
+            same shape, except in the horizontal (column wise) dimension.
```

Comment:
So, sorry... I have just recognized what you mean.

How about this one? I will rewrite like this.

<img width="596" alt="2017-04-20 17 28 09" src="https://cloud.githubusercontent.com/assets/1277089/25221111/eb05fe22-25ee-11e7-9d21-561892b766f5.png">